guides/plugins/plugins/administration/customizing-modules.md

```diff
@@ -0,0 +1,56 @@
+# Customize modules
+
+## Overview
+
+In the `Administration` core code, each module is defined in a directory called `module`.
+A `module` is an encapsulated unit which implement a whole feature. For example there are modules for customers, orders, settings, etc.
+
+## Prerequisites
+
+All you need for this guide is a running Shopware 6 instance. Of course, you'll have to understand JavaScript and have a basic familiarity with TwigJS, the templating engine, used in the Administration. However, that's a prerequisite for Shopware as a whole and will not be taught as part of this documentation.
+
+## Customizing a module
+
+Module settings like `color`, `icon`, `navigation` are fixed by design and cannot be changed.
+
+A guide for customizing components which are already defined in existing modules can be found here - [Customizing components](./customizing-components.md).
+
+At some point you need add or change the routes of a module. For example when you want to add a tab to the page
+
```

Comment:
Missing something like: "However, modules themselves cannot be directly overriden." or smth like that.

Reply:
added your suggestion